mod_app/templates/film_list.html

```diff
@@ -7,26 +7,37 @@
         List of films on this site
     </div>
 
-    <div class="card-area">
-        {% for film in film_list %}
-            <div class="card card--film">
-                <h4 class="card__heading"><a href="{% url 'film_detail' pk=film.pk %}">{{film.title}}</a></h4>
-                <div class="card__info">
-                    <dl>
-                        <dt>Starring:</dt>
-                        <dd>{{film.starring}}</dd>
-                        <dt>Release:</dt>
-                        <dd>{{film.release_date}}</dd>
-                        <dt>Themes:</dt>
-                        <dd>{{film.themes.all}}</dd>
-                        <dt>Genre:</dt>
-                        <dd class="tag"><span class="dot">•</span>{{genre.name}}</dd>
-                        <dt>Summary:</dt>
-                        <dd>{{film.summary|truncatewords:50}}</dd>
-                    </dl>
-                    <button href="{% url 'film_detail' pk=film.pk %}"> view full details </button>
-                </div>
-            </div>
-        {% endfor %}
-    </div>
+    <table class="film-table">
+        <thead>
+            <tr>
+                <th>Title</th>
+                <th>Starring</th>
+                <th>Release Date</th>
+                <th>Genre</th>
+                <th>Summary</th>
+                <th>Go to page</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for film in film_list %}
+                <tr>
+                    <td><a href="{% url 'film_detail' pk=film.pk %}">{{film.title}}</a></td>
+                    <td>{{film.cast}}</td>
+                    <td>{{film.release_date}}</td>
+                    <td>
+                        {% for g in film.genre.all %}
+                            <span class="tag"><span class="dot">•</span>{{g.name}}</span>
+                        {% endfor %}
+                    </td>
+                    <td>{{film.summary|truncatewords:50}}</td>
+                    <td><a href="{% url 'film_detail' pk=film.pk %}" target="_blank">
```

Comment:
Why open in new tab? Apparently this is worse for accessibility.

Reply:
Oh, I think that's my internal bias coming out since I always open things in new tabs (and also that's what the icon indicates) - I can change the behaviour and the icon if that would be better?